deposit/bytecode-verification/deposit-spec.ini

```diff
@@ -1346,17 +1347,17 @@ status_code: _ => EVMC_REVERT
 ; Revert if the first four bytes are invalid (or not fully provided).
 [revert-invalid_function_identifier]
 call_data: #buf(CALL_DATA_SIZE, CALL_DATA)
-pc: 0 => 4345
+pc: 0 => 4277
 gas:         #symGas(G, 0 => {GAS_COST}, 0 => {GAS_COST}, .List, Cmem({SCHEDULE}, {MEMORY_USED}))
 memory_used: #symMem(0 => {MEM_COST}, .Set)
 
 [revert-invalid_function_identifier-lt_4]
 +requires:
     // conditions
     andBool #range(0 <= CALL_DATA_SIZE < 4)
-    andBool #asWord(#buf(CALL_DATA_SIZE, CALL_DATA) ++ #buf(4 -Int CALL_DATA_SIZE, 0)) =/=Int 3321006383 /* 0xc5f2892f get_deposit_root */
-    andBool #asWord(#buf(CALL_DATA_SIZE, CALL_DATA) ++ #buf(4 -Int CALL_DATA_SIZE, 0)) =/=Int 1646252336 /* 0x621fd130 get_deposit_count  */
-    andBool #asWord(#buf(CALL_DATA_SIZE, CALL_DATA) ++ #buf(4 -Int CALL_DATA_SIZE, 0)) =/=Int 579424536  /* 0x22895118 deposit            */
+;   andBool #asWord(#buf(CALL_DATA_SIZE, CALL_DATA) ++ #buf(4 -Int CALL_DATA_SIZE, 0)) =/=Int 3321006383 /* 0xc5f2892f get_deposit_root */
```

Comment:
This case is dealt with below. Let's remove these lines instead of commenting them out?